templates/pulpcore-resource-manager.service.erb

```diff
@@ -0,0 +1,27 @@
+[Unit]
+Description=Pulp Resource Manager
+After=network-online.target
+Wants=network-online.target
+
+[Service]
+Environment="DJANGO_SETTINGS_MODULE=pulpcore.app.settings"
+Environment="PULP_SETTINGS=/etc/pulp/settings.py"
+Environment="PATH=/usr/local/lib/pulp/bin:"
+User=pulp
+WorkingDirectory=/var/run/pulpcore-resource-manager/
+RuntimeDirectory=pulpcore-resource-manager
+ExecStart=/usr/local/lib/pulp/bin/rq worker \
+          -w pulpcore.tasking.worker.PulpWorker -n resource-manager@%%h \
+          --pid=/var/run/pulpcore-resource-manager/resource-manager.pid \
```

Comment:
https://python-rq.org/patterns/systemd/ also suggests to rely on systemd's `$MAINPID`.